item-source.js

```diff
@@ -0,0 +1,40 @@
+export const _getLength = new WeakMap();
+export const _item = new WeakMap();
+export const _key = new WeakMap();
```

Comment:
I guess I should just use symbols since that's what the rest of the code is using. Let me switch it.